bmc-log-collector/main.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"log"
+	"log/slog"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+)
+
+type bmcLogWriter interface {
+	write(stringJson string, serial string) (err error)
+}
+
+func doLogScrapingLoop(config selCollector, logWriter bmcLogWriter) {
+	var wg sync.WaitGroup
```

Comment:
Please move this variable definition. It is better to define just before use.

Reply:
Moved it.